docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,63 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work, please create an issue if you run into problems during the
+migration to Antrea. In addition, during the migration process, no Kubernetes
+resources should be created or deleted, otherwise the migration process might
+fail or some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up old CNI.
+2. Install Antrea in the cluster.
+3. Restart all Pods in the cluster in-place.
+
+## Clean up old CNI
+
+Before installing Antrea in the cluster, the old CNI needs to be cleaned up. The
+cleanup process is different for different CNIs, but the basic idea is to
+remove the old CNI DaemonSet, binaries and configuration files from the cluster.
+If the old CNI use iptables to manage the network, the iptables rules need to be
+cleaned up as well.
+
+## Install Antrea
+
+The first step is to install Antrea in the cluster. You can follow the
+[installation guide](https://github.com/antrea-io/antrea/blob/main/docs/getting-started.md)
+to install  Antrea in the cluster. The following is an example of
+installing Antrea with `build/yamls/antrea.yml`:
+
+```bash
+$ kubectl apply -f build/yamls/antrea.yml --kubeconfig ~/.kube/config
+```
```

Comment:
We don't recommend users to deploy the in-development version, and don't add the unnecessary kubeconfig parameter which may just cause the command don't work when users use different path. They don't need us to tell how to load kube config.

Just use the first method:
The following command deploys Antrea with the version specified by `<TAG>`(e.g. v1.14.1)
```
kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
```

Reply:
Agree, an example added.